sysbuild/CMakeLists.txt

```diff
@@ -390,6 +390,11 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
         set_config_bool(${DEFAULT_IMAGE} CONFIG_CHIP_OTA_REQUESTOR n)
         set_config_bool(${DEFAULT_IMAGE} CONFIG_CHIP_OTA_IMAGE_BUILD n)
       endif()
+      # Set default SUIT envelope templates paths for application and radio images for Matter
+      if(SB_CONFIG_SUIT_ENVELOPE AND SB_CONFIG_MATTER_USE_COMMON_SUIT_ENVELOPE_TEMPLATES)
```

Comment:
You don't have to modify this file, it was already done by Arek in his PR (we just need to wait for his change to be merged first).